src/components/carousel/carousel-items.jsx

```diff
@@ -22,21 +23,23 @@ function CarouselItem(props) {
           borderRadius: "25px",
         }}
       >
-        <Box
-          sx={{
-            ...imageContainerStyles,
-          }}
-        >
-          <ProductImage src={props.item.imgPath} width="100%" height={300} />
-          <CategoryItems
-            title={props.item.name}
-            description={props.item.description}
-          />
-        </Box>
+        <Link href="#">
```

Comment:
why are you wrapping the whole carousel with a link, not the whole image should be clickable but only the "see more" button